parser/intent.go

```diff
@@ -73,11 +77,24 @@ func ExpectedOperations(
 				continue
 			}
 
-			if ExpectedOperation(in, obs) == nil {
-				matches[i] = struct{}{}
-				foundMatch = true
-				break
+			err := ExpectedOperation(in, obs)
+			if err != nil {
+				continue
```

Comment:
Do we not care about this error? Looks like it's not being logged or returned anywhere

Reply:
@slowboat0 just updated code with a comment around this to explain!